localization/english/wc_game_concepts_l_english.yml

```diff
@@ -83,10 +83,10 @@
 
  game_concept_scourge:0 "Scourge"
  game_concept_scourge_possessive:0 "Scourge's"
- game_concept_scourge_desc:0 "One of the main threats on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nIt enables [great_invasion_casus_belli|E] but instead of taking lands you take vassals and turn them [GetTrait('being_undead').GetName( GetNullCharacter )].\n\nThe $game_concept_scourge$ starts as a [legions_puppet|E] thus the first 15 years are dedicated to bringing the Legion to Azeroth."
+ game_concept_scourge_desc:1 "One of the main [crises|E] on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nIt enables [great_invasion_casus_belli|E] but instead of taking lands you take vassals and turn them [GetTrait('being_undead').GetName( GetNullCharacter )].\n\nThe $game_concept_scourge$ starts as a [legions_puppet|E] thus the first 15 years are dedicated to bringing the Legion to Azeroth."
```

Comment:
```suggestion
 game_concept_scourge_desc:1 "One of the main [crises|E] on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nThe $game_concept_scourge$ has the [great_invasion_casus_belli|E] ,but instead of taking lands it takes vassals and turn them [GetTrait('being_undead').GetName( GetNullCharacter )].\n\nThe $game_concept_scourge$ also starts as a [legions_puppet|E], and must spend the first 15 years dedicated to bringing the Legion to Azeroth."
```